resources/help.txt

```diff
@@ -112,10 +112,21 @@ Usage:
 
 Platform-specific usage:
     $ tns platform add android [--frameworkPath <File Path>] [--symlink]
-    $ tns platform add ios [--frameworkPath <File Path>] [--symlink] 
+    $ tns platform add android[@<Version>] [--frameworkPath <File Path>] [--symlink]
+    $ tns platform add ios [--frameworkPath <File Path>] [--symlink]
+    $ tns platform add ios[@<Version>] [--frameworkPath <File Path>] [--symlink]
 
 Configures the current project to target the selected platform. 
 
+<Version> is any available version published in npm.
```

Comment:
Let's drop the marketing talk from here. Instead, I'll try incorporating some of it in the readme and the changelog. I'll put up a separate PR for it.

In addition, please, transfer this to the update command as well.